### ISSUE TITLE
How to add list to Marquee

### ISSUE THREAD

**Issue by almog228** [marcelgarus/marquee#72]
Hello I have a number of messages that are in a list. I want to display them one after the other but he lets me insert only a string. Is there another way to display the list line by line?

**Comment by marcelgarus** [comment]
Sadly, that's not possible. If you just want to display multiple messages one after another, you can of course separate them with spaces, but I suppose you also want to stop at every message?

**Comment by almog228** [comment]
Yes

בתאריך יום ב׳, 27 בספט׳ 2021, 11:50, מאת Marcel Garus ‏<
***@***.***>:

**Comment by marcelgarus** [comment]
Yeah, that would require a major re-architecture of the widget. ☹️ See #23 for more details
